WordPress/Classes/ViewRelated/Stats/Charts/DonutChartView.swift

```diff
@@ -39,7 +39,7 @@ class DonutChartView: UIView {
 
     var totalCount: CGFloat = 0 {
         didSet {
-            totalCountLabel.text = Float(totalCount).abbreviatedString()
+            totalCountLabel.text = Double(totalCount).abbreviatedString()
```

Comment:
This type change is because the previous implementation defined the method on a `Float` extension but used `Double` under the hood. 

Definitely clearer to have the extension on `Double`.